jito_searcher_client/searcher.py

```diff
@@ -14,6 +11,10 @@
 from grpc.aio import ClientCallDetails
 from solders.keypair import Keypair
 
+from jito_searcher_client.generated.auth_pb2_grpc import AuthServiceStub
+from jito_searcher_client.generated.searcher_pb2_grpc import SearcherServiceStub
+from jito_searcher_client.token import JwtToken
+
 from .generated.auth_pb2 import (
```

Comment:
nit: use full import
```suggestion
from jito_searcher_client.generated.auth_pb2 import (
```